launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
I ended up adding docs regarding the use of "npm link" to add a "flocking-midi-router" command to your path.